fivetran_provider_async/operators.py

```diff
@@ -1,13 +1,92 @@
 from typing import Any, Dict, Optional
 
 from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator, BaseOperatorLink
 from airflow.utils.context import Context
-from fivetran_provider.operators.fivetran import FivetranOperator
+from airflow.utils.decorators import apply_defaults
 
+from fivetran_provider_async.hooks import FivetranHook
 from fivetran_provider_async.triggers import FivetranTrigger
 from fivetran_provider_async.utils.operator_utils import datasets
 
 
+class RegistryLink(BaseOperatorLink):
+    """Link to Registry"""
+
+    name = "Astronomer Registry"
+
+    def get_link(self, operator, dttm):
+        """Get link to registry page."""
+
+        registry_link = "https://registry.astronomer.io/providers/{provider}/modules/{operator}"
+        return registry_link.format(provider="fivetran", operator="fivetranoperator")
+
+
+class FivetranOperator(BaseOperator):
```

Comment:
Should we merge `FivetranOperator` and `FivetranOperatorAsync` as one operator as OSS airflow?

Reply:
@phanikumv wdyt ^^?

Reply:
we shouldnt put anything related to Fivetran into OSS airflow for the moment

Reply:
oh, what I mean is merging them into one operator in this project

Reply:
yes on this repo let's do it

Reply:
I like the idea of merging the operators in OSS

Reply:
+1

Reply:
then would we deprecate the `FivetranOperatorAsync` operator? 
I'll say for the scope of this PR let's just move the code from the original repo to this repo and in a separate PR we can merge both FivetranOperatorAsync and FivetranOperator and deprecate FivetranOperatorAsync wdyt?

Reply:
thats fine as well

Reply:
yes +1 to deprecating FivetranOperatorAsync and merging in FivetranOperator with deferrable flag. Good idea

Reply:
created an issue https://github.com/astronomer/airflow-provider-fivetran-async/issues/35